.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,6 +12,6 @@ _Fixes #_
 
 ### Pre-checks (if applicable):
 - [ ] Tested latest changes against a cluster
-- [ ] Ran `make test-unit` command locally to run all the unit tests and mock tests locally.
+- [ ] Ran `make go-test` command locally to run all the unit tests and mock tests locally.
 - [ ] Included documentation changes with PR
 - [ ] Squashed your commits
```

Comment:
Unrelated - As we're currently updating the template, it would be best to remove this line. When it comes to committing changes, it is up to the engineer's discretion whether to squash them or not. Having individual commits can facilitate better code reviewing. However, we can still use a label to force Prow to squash commits.

Reply:
Sounds fine to me, I've removed it.